File: Modul 7/soal 3/PRAK603-2210817220020-Helena Rosalina.c
#include <stdio.h>
int main()
{

int baris1,baris2,i,isi,hasil;
scanf("%d %d", &baris1, &baris2);
int bilangan1[baris1],bilangan2[baris2];
    if (baris1==baris2){
        for(i=0;i<baris1;i++){
        scanf("%d", &isi);
        bilangan1[i] = isi ;
        }
        for(i=0;i<baris2;i++){
        scanf("%d", &isi);
        bilangan2[i] = isi ;
        }
printf("\n");
        for(i=0;i<baris1;i++){
        hasil=bilangan1[i] * bilangan2[i];
        printf("%d ", hasil);
        }
    }
    else{
    printf("Jumlah Tidak Sama");
    }
return 0;
}

File: Modul 7/soal 3/PRAK603-2210817220020-Helena Rosalina.py
baris1,baris2=map(int,input().split())
if baris1!=baris2:
    print('Jumlah Tidak Sama')
else:
    bilangan1= list(map(int,input().split()))
    bilangan2= list(map(int,input().split()))
    matriks1=[]
    matriks2=[]
    x=0
    matriks1.append(bilangan1[x:x + baris1])
    matriks2.append(bilangan2[x:x + baris2])
    for i in range(0,baris1):
        print(matriks1[0][i]*matriks2[0][i], end=' ')